src/main/java/yapper/storage/Storage.java

```diff
@@ -0,0 +1,68 @@
+package yapper.storage;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import yapper.task.Task;
+import yapper.task.Todo;
+import yapper.task.Deadline;
+import yapper.task.Event;
+import yapper.exception.YapperException;
+
+public class Storage {
+    private final String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public ArrayList<Task> load() throws IOException {
```

Comment:
The methods load() and save() could benefit from Javadoc comments.

Reply:
sure